internal/cluster/transition.go

```diff
@@ -469,6 +475,22 @@ func (th *transitionHandler) InstallCluster(sw stateswitch.StateSwitch, args sta
 	if err := params.dnsApi.CreateDNSRecordSets(ctx, cluster); err != nil {
 		return err
 	}
+	if common.IsSingleNodeCluster(cluster) {
```

Comment:
this code should be part of transition from preparing for installation to installing. 
This is what we talked about previously, oir i miss something?